datasets/mx/tax_offender/crawler.py

```diff
@@ -0,0 +1,124 @@
+import csv
+from typing import List
+
+from zavod import Context, helpers as h
+
+URLs = [
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Cancelados.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/ReduccionArt74CFF.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Condonadosart146BCFF.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Condonadosart21CFF.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/CondonadosporDecreto.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Condonados_07_15.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Cancelados_07_15.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Retornoinversiones.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Exigibles.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Firmes.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/No%20localizados.csv",
+    "http://omawww.sat.gob.mx/cifras_sat/Documents/Sentencias.csv",
+]
+
+
+def replace_key(d: dict, possible_old_keys: List[str], new_key: str) -> dict:
+    """
+    This function returns the dictionary d with the first key found in possible_old_keys
+    replaced by the new key
+    """
+
+    for possible_old_key in possible_old_keys:
+        if possible_old_key in d:
+            d[new_key] = d.pop(possible_old_key)
+            break
+    return d
+
+
+def crawl_item(input_dict: dict, context: Context):
+
+    schema = "Person" if input_dict.pop("person_type") == "F" else "Company"
+
+    entity = context.make(schema)
+    entity.id = context.make_id(input_dict.get("RFC"), input_dict.get("name"))
+    entity.add("name", input_dict.pop("name"))
+    entity.add("taxNumber", input_dict.pop("RFC"))
+    entity.add("topics", "debarment")
```

Comment:
I think this should be `reg.action` but let's wait for confirmation

```suggestion
    entity.add("topics", "reg.action")
```